### PR TITLE
docs: fix guildChannel#toString() example

### PR DESCRIPTION
**Pull request by devdutchy**
**Please describe the changes this PR makes and why it should be merged:**
self-explanatory, see #2201 

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/GuildChannel.js [hunk 1]
```diff
@@ -399,10 +399,10 @@ class GuildChannel extends Channel {
    * When concatenated with a string, this automatically returns the channel's mention instead of the Channel object.
    * @returns {string}
    * @example
-   * // Outputs: Hello from #general
+   * // Logs: Hello from <#123456789012345678>!
    * console.log(`Hello from ${channel}`);
```
Comment (bdistin) [inline src/structures/GuildChannel.js]: This doesn't include a `!` like you say it should log.

File: src/structures/GuildChannel.js [hunk 2]
```diff
@@ -399,10 +399,10 @@ class GuildChannel extends Channel {
...
    * @example
-   * // Outputs: Hello from #general
+   * // Logs: Hello from <#123456789012345678>!
    * console.log('Hello from ' + channel);
```
Comment (bdistin) [inline src/structures/GuildChannel.js]: same

File: src/structures/GuildChannel.js (hunk 1 above)
Comment (devdutchy) [reply src/structures/GuildChannel.js]: Fixed.

**Comment by 2E0PGS** [comment]
I approve.

**Pull request merged**